app/src/androidTest/java/org/dhis2/usescases/form/FormTest.kt

```diff
@@ -175,6 +175,7 @@ class FormTest: BaseTest() {
         }
 
         enrollmentRobot {
+            waitToDebounce(500)
```

Comment:
So, now with the idling, this should be removed, righ?

Reply:
my bad! I thought I removed everything